packages/office-ui-fabric-react/src/components/HoverCard/HoverCard.tsx

```diff
@@ -139,7 +154,7 @@ export class HoverCard extends BaseComponent<IHoverCardProps, IHoverCardState> {
 
     this._dismissTimerId = this._async.setTimeout(() => {
       if (!(this.props.sticky && type === 'mouseleave')) {
-        const rect = this._hoverCard.getBoundingClientRect();
+        const rect = this._expandingCard.element.getBoundingClientRect();
```

Comment:
Is there a better way than to measure this? For example, when the cursor goes over it, set a state to true, when it leaves, set a state to false, then never measure? Measuring is very expensive.